kubetest/e2e.go

```diff
@@ -99,6 +100,18 @@ func run(deploy deployer, o options) error {
 		}))
 	}
 
+	if o.postTestCmd != "" {
+		// Guaranteed to run after the entire test, including teardown.
+		defer control.XMLWrap(&suite, "Deferred post-test command", func() error {
+			if o.test || (kubemarkUpErr == nil && o.testCmd != "") {
```

Comment:
Would it be okay if we simply use the `o.test || o.testCmd != ""` condition?

Reply:
I'm not entirely sure of the context of kubemarkUpErr check here, so I preserved the conditions. @marseel can you help with this?